tests/integration/IntegrationTest.js

```diff
@@ -0,0 +1,107 @@
+'use strict'
+
+const tape = require('tape')
+const http = require('http')
+const supertest = require('supertest')
+const {createTestEndpoint, removeTestEndpoint} = require('../testUtils')
+
+// Create an endpoint for the tests
+createTestEndpoint()
+
+const app = require('../../src/index')
+
+// Remove the test endpoint folder
+removeTestEndpoint()
+
+tape.test('Parsing Integration Tests', t => {
+  tape.onFinish(() => {
+    server.close()
+  })
+
+  const server = http.createServer(app.callback())
```

Comment:
I think we can define this within the before/after methods of test so that it runs before all the tests and after all the tests
```
test.before("before all tests", function (assert) {
    // startup
})

test.after("after all tests", function (assert) {
    // cleanup
})
```

Reply:
tape does not have the before and after hooks

Reply:
Ok cool, lets maybe try this approach then instead
https://github.com/substack/tape/issues/59#issuecomment-32809481

Reply:
Maybe we should replace using `tape` with actual `tap` which also includes a few features making it easier to use/view.

Reply:
Have implemented this :+1: